src/main/java/seedu/address/model/student/Homework.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.model.student;
+
+import java.time.LocalDate;
+
```

Comment:
```suggestion

/**
 * Represents an assignment that is of type "homework" and extends from Assignment class.
 *
 * @author mayuanxin1234
 * @since (add date)
 */
```